operations/mimir-mixin/dashboards/dashboard-utils.libsonnet

```diff
@@ -701,20 +701,25 @@ local utils = import 'mixin-utils/utils.libsonnet';
       [
         |||
           sum by (scaler) (
-            label_replace(
-              # Using `max by ()` so that series churn doesn't break the promQL join
-              max by (%(aggregation_labels)s, scaledObject, metric, scaler) keda_scaler_metrics_value{%(cluster_label)s=~"$cluster", exported_namespace=~"$namespace", scaler=~"%(scaler_selector)s"},
-              "namespace", "$1", "exported_namespace", "(.*)"
-            )
-            /
-            on(%(aggregation_labels)s, scaledObject, metric) group_left label_replace(
+            # Using `max by ()` so that series churn doesn't break the promQL join
+            max by (%(aggregation_labels)s, scaledObject, metric, scaler) (
```

Comment:
should this also include `namespace`?

Reply:
namespace is already part of the aggregation_labels